### PR TITLE
Add extension onboarding pages

### PR DESCRIPTION
**Pull request by cquintana92**
This PR performs the following changes:

1. Adds new onboarding pages to be used with the extension. Both the layout and the behaviour are implemented.
2. Adds the Proton partner with the `flask dummy-data` command.

### CODE REVIEW

File: app/onboarding/views/final.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    email = StringField("Email", validators=[validators.DataRequired()])
+
+
+@onboarding_bp.route("/final", methods=["GET", "POST"])
+@login_required
+@limiter.limit("10/minute")
+def final():
+    form = SendEmailForm(request.form)
+    if form.validate_on_submit():
+        send_test_email_alias(form.email.data, current_user.name)
```
Comment (nguyenkims) [inline app/onboarding/views/final.py:20]: That's a good idea to use the existing `send_test_email_alias` here 👌. I think we need to update the wording in the email to remove the "If you have any questions, feel free to reply to this email." part as now all emails are sent from a no-reply address.

File: app/onboarding/views/final.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        flash("We have sent a test e-mail to your alias", "success")
+
+    return render_template(
+        "onboarding/final.html",
```
Comment (nguyenkims) [inline app/onboarding/views/final.py:24]: I think the page misses some elements
- an indication for users to click on the SL icon to create a new alias
- an indication that user can also right click to create a new alias
- a link at bottom to the web dashboard

image

**Review by nguyenkims — COMMENTED** [review]
The PR looks good to me, just left some comments.

File: app/onboarding/views/final.py (hunk 1 above)
Comment (cquintana92) [reply app/onboarding/views/final.py:20]: We could change the line by

```
If you have any questions, feel free to contact us either by sending an e-mail to <a href="mailto:hi@simplelogin.io">hi@simplelogin.io</a) or through our <a href="/dashboard/support">support page</a>
```

Comment (cquintana92) [reply app/onboarding/views/final.py:20]: Pushed an initial implementation. Let me know what you think.

File: templates/emails/transactional/test-email.txt [hunk 3]
```diff
@@ -5,7 +5,7 @@ Hi {{name}}
 
 This is a test to make sure that you receive emails sent to your alias {{alias}}.
 
-If you have any questions, feel free to reply to this email.
+If you have any questions, feel free to contact us either by sending an e-mail to {{support_email}} or through our support page: {{support_page}}
```
Comment (nguyenkims) [inline templates/emails/transactional/test-email.txt:8]: Ah actually we don't want to put the support email here as we want to encourage users to use the support page instead.

Comment (cquintana92) [reply templates/emails/transactional/test-email.txt]: Removed it

File: app/onboarding/views/final.py (hunk 2 above)
Comment (cquintana92) [reply app/onboarding/views/final.py:24]: I've added the link at the bottom to the web dashboard. I'll try to finish the indicators for the users to click either on the extension icon or to use the right click

File: app/onboarding/views/final.py (hunk 1 above)
Comment (acasajus) [reply app/onboarding/views/final.py:20]: I would remove the part about sending us and email and reference them to use our support channel.

Comment (cquintana92) [reply app/onboarding/views/final.py:20]: Replaced it with

HTML:
```
{{ render_text('If you have any questions, feel free to contact us through our <a href="' + support_page +'">support page</a>') }}
```

TXT:
```
If you have any questions, feel free to contact us through our support page: {{support_page}}
```

File: templates/emails/transactional/test-email.html [hunk 4]
```diff
@@ -3,6 +3,6 @@
 {% block content %}
   {{ render_text("Hi " + name) }}
   {{ render_text("This is a test to make sure that you receive emails sent to your alias <b>" + alias + "</b>.") }}
-  {{ render_text("If you have any questions, feel free to reply to this email.") }}
```
Comment (nguyenkims) [inline templates/emails/transactional/test-email.html:6]: I think we can remove this sentence as the support page is already mentioned in the email footer:

image

File: templates/emails/transactional/test-email.txt [hunk 5]
```diff
@@ -5,7 +5,7 @@ Hi {{name}}
 
 This is a test to make sure that you receive emails sent to your alias {{alias}}.
 
-If you have any questions, feel free to reply to this email.
+If you have any questions, feel free to contact us through our support page: {{support_page}}
```
Comment (nguyenkims) [inline templates/emails/transactional/test-email.txt:8]: We can remove this sentence as the support is also mentioned in the text email

image

**Review by nguyenkims — APPROVED** [review]
LGTM! We can just remove the last sentence in the test email and then merge the PR :).

File: templates/emails/transactional/test-email.html (hunk 4 above)
Comment (cquintana92) [reply templates/emails/transactional/test-email.html:6]: Removed it

File: templates/emails/transactional/test-email.txt (hunk 5 above)
Comment (cquintana92) [reply templates/emails/transactional/test-email.txt]: Removed it too

**Pull request merged**